pyrdp/player/RDPMITMWidget.py

```diff
@@ -197,4 +197,4 @@ def sendCurrentScreen(self):
         width = self._buffer.width()
         height = self._buffer.height()
         pdu = PlayerBitmapPDU(self.layer.getCurrentTimeStamp(), width, height, self._buffer.bits())
-        self.layer.sendPDU(pdu)
\ No newline at end of file
+        self.layer.sendPDU(pdu)
```

Comment:
Try to avoid unrelated changes, use `git add -p` (or the GUI equivalent) instead of `git add` to select what you commit.

Not for this PR but for the next one

Reply:
Yeah, will do.